scripts/categories.rb

```diff
@@ -1,4 +1,7 @@
-#!/usr/bin/ruby
```

Comment:
@brucellino, a script without a shebang looks a bit weird, especially as the documentation for running it is missing. It's also best to use /usr/bin/env to find the appropriate ruby location.

Reply:
I can live with that. We should probably have a bit of that in a style guide somewhere : 
> All scripts should have a shebang
good point on the documentation... I'll send a new PR to fix this, since the branch has been deleted... SSH is actually hard, but this is a style problem.